src/data/init/init-moves.ts

```diff
@@ -3259,7 +3259,7 @@ export function initMoves() {
       .attr(TeraBlastTypeAttr)
       .attr(TeraBlastPowerAttr)
       .attr(StatStageChangeAttr, [Stat.ATK, Stat.SPATK], -1, true, {
-        condition: (user, _target, _move) => user.isTerastallized() && user.isOfType(ElementalType.STELLAR),
+        condition: (user, _target, _move) => user.isTerastallized && user.isOfType(ElementalType.STELLAR),
```

Comment:
NIT: If you dont' need params after the optionals you can omit them:
```suggestion
        condition: (user) => user.isTerastallized && user.isOfType(ElementalType.STELLAR),
```